src/main/java/prioneer/homework/member/web/admin/system/UserNewController.java

```diff
@@ -28,17 +30,11 @@ public String showNewUserForm(Model model) {
     @PostMapping("/system/user")
     public String registerNewUser(@ModelAttribute("member") Member member, RedirectAttributes redirectAttributes) {
         try {
-            member.setRole("member");
-            member.setDepositDepend(0L); // 초기 보증금 방어권 0개
-            member.setDeposit(120000L); // 초기 보증금 12만원
-
-            memberRepository.save(member);
-
-            redirectAttributes.addFlashAttribute("message", "회원 등록 성공!");
-            return "redirect:/system/user";
-        } catch (Exception e) {
-            redirectAttributes.addFlashAttribute("message", "회원 등록 실패");
-            return "redirect:/system/user";
+            adminMemberService.registerNewMember(member);
+            redirectAttributes.addFlashAttribute("message", "회원 등록 성공");
+        } catch (IllegalStateException e) {
```

Comment:
redirectAttributes.addFlashAttribute 흠 이런 부분은 없어도 될거같아요!
@Valie BindResult 해서 오류메시지 담아서 프론트에 보낼 수 있습니다.


```suggestion
    @PostMapping("/login")
    public String postLogin(@Validated @ModelAttribute Member member, BindingResult bindingResult,
                            @RequestParam(defaultValue = "/") String redirectURL,
                            HttpServletRequest request) throws Exception {

        Optional<Member> findLoginMember = loginService.loginIdCheck(member);
        if (findLoginMember.isEmpty()) {
            bindingResult.reject("loginFail", "아이디가 존재하지 않습니다");
            return "member/login";
        }

        Member passwordMember = loginService.passwordCheck(member);
        if (passwordMember == null) {
            bindingResult.reject("loginFail", "비밀번호가 일치하지 않습니다");
            return "member/login";
        }

        //로그인 성공
        HttpSession session = request.getSession();
        session.setAttribute(SessionConst.LOGIN_MEMBER, passwordMember);

        return "redirect:" + redirectURL;
    }
    
```
    
    여기 제  예전 코드 예시본 드립니다.